package.json

```diff
@@ -74,6 +74,12 @@
           "default": "credentials.yml",
           "description": "Ansible credential file path",
           "scope": "resource"
+        },
+        "ansible.terminalInitCommand": {
```

Comment:
this looks quite advanced feature. if user would like to set this configuration, they need clear understanding on workspace context, docker image name. what's usage scenario for this configuration? run playbook in docker by terminal? authoring? develop?

Reply:
that's why we should keep it hidden at the moment, but we should design our extension around configuration like that. instead of having if (...) "docker run" else if (...) "cloudshell .." etc.

"run playbook in docker by terminal" will just change to "run playbook in terminal" which will be generic. terminal should be in appropriate state to run the playbook, in case of docker:
- current directory should be mapped to running docker
- current directory should be set to that mapped directory
so "run playbook in terminal" will just send "ansible-playbook .... ./yourplaybook.yml" to terminal transparently without referring to docker.

Reply:
in case of cloudshell it may be a bit more complex (we can't map current directory), but perhaps we can send:
echo "playbook content" > ./playbook.yml ; ansible-playbook ... ./playbook.yml

still very generic.